<launch>

  <!-- include file="$(find jr_navigation)/launch/jr_nav_server.launch" / -->
  
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>
  
  <node name="recognizer" pkg="pocketsphinx" type="recognizer.py" output="screen">
    <param name="lm" value="$(find jr_speech)/config/3dv_demo/3dv_commands.lm"/>
    <param name="dict" value="$(find jr_speech)/config/3dv_demo/3dv_commands.dic"/>
  </node>

  <node pkg="jr_speech" type="jr_demo.py" name="jr_demo" clear_params="true" output="screen">
     <remap from="target_topic" to="/face_recognizer/face_recognitions" />
     
     <param name="nav_config_file" value="$(find jr_navigation)/config/locations.yaml" />
     
     <rosparam>
        rate: 10
        max_target_distance: 2.0
        lost_detection_timeout: 5.0
        tts_voice: voice_don_diphone
     </rosparam>
  </node>
   
</launch>
